templates/adherent/profile.html.twig

```diff
@@ -10,6 +16,10 @@
         Kernel.onLoad(function() {
             {{ js.address(form.address, { cityAutocomplete: true }) }}
         });
+
+        $(document).ready(function() {
```

Comment:
IMO we can call the ` $('.select2').select2();` inside the `Kernel.onLoad` no ?

Reply:
:heavy_check_mark: